src/backend/feed/worker.js

```diff
@@ -5,16 +5,19 @@ const { logger } = require('../utils/logger');
 
 const Post = require('../post');
 
+const sanitizeHTML = require('../utils/sanitize-html');
+
 exports.workerCallback = async function(job) {
   const { url } = job.data;
   const posts = await feedparser(url);
   const processedPosts = await Promise.all(
     posts.map(async post => {
       // TODO: run this through text parser and sanitizer
```

Comment:
Update this comment, since you're implementing the latter portion of it.

Reply:
Comment has been updated.